debezium-core/src/test/java/io/debezium/util/LruCacheTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.util;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import org.junit.Test;
+
+/**
+ * Unit test for {@link LruCache}.
+ *
+ * @author Gunnar Morling
+ */
+public class LruCacheTest {
+
+    @Test
+    public void shouldRemoveEldestEntry() {
```

Comment:
The test is not complete. It verifies that the oldest written element has been removed, nto the last accessed which would represent truly LRU.

Reply:
Added a test. It's the former QE in you coming through :)